src/Library/demos/Navigation View/main.blp

```diff
@@ -0,0 +1,90 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  Box {
+    halign: center;
+    valign: start;
+    orientation: vertical;
+    Box {
+      height-request: 360;
+      width-request: 360;
+      halign: center;
+      margin-bottom: 24;
+      orientation: vertical;
+      styles ["card"]
+
+      Adw.NavigationView nav_view {
+        Adw.NavigationPage nav_pageone {
+          title: _("Page 1");
+          Adw.StatusPage {
+            title: _("Hello 👋");
+            description: _("➡️");
```

Comment:
These should probably just be standard labels (of some sort), especially considering this widget is probably sensitive to LTR vs RTL language locales.